script.js

```diff
@@ -2,49 +2,73 @@
 //Declare a variable called 'name' that has the value of your name.
 
 /*code here*/
-
+var name="Jose";
 //#2
 //create an if statement that checks to see if your name is equal to 'Ben'
 //If yes, log 'yes'
 //otherwise log 'no'
 //note: use console.log to log the string
 
 /*code here*/
-
+if( name==='Ben')
+  Console.log("yes");
+else 
+  Console.log("no");
 //#3
 //create an array called 'favoriteFoods'.
 //fill it up with the names of several of your favorite foods
 
 /*code here*/
-
+var favoriteFoods=[
+  "Bananas","Apples","oranges","tika masala","pears", "California Burritos", "loquats"
+];
 //#4
 //use a for loop to log each food in the 'favoriteFoods' array
 
 /*code here*/
-
+for(var i=0;i<favoriteFoods.lenght;++i){
+  console.log(favoriteFoods[i]);
+}
 //#5
 //create an object called 'favoriteMovie'.
 //give 'favoriteMovie' a property called 'runtime' and set it equal to how long the movie is in minutes
 //give 'favoriteMovie' a property called 'title' and set it equal to the title
 //give 'favoriteMovie' a property called 'director' and set it equal to the director's name
 
 /*code here*/
-
+var favoriteMovie={
+  runtime:"1h 08m",
+  title:"BuenaVistaSocialClub",
+  director:"Wim Wenders"
 //#6
 //create a function called 'sayHi'.
 //'sayHi' should accept one argument called 'name'
 //when 'sayHi' is invoked it should log the string 'Hello <name>!' where <name> is equal to the 'name' argument
 
 /*code here*/
-
+function sayHi(Jose){
+  console.log("Hello"+Jose+"!");
+};
```

Comment:
If you were to write a function expression like this then you would want a semicolon -> `var sayHi = function(){};`